test/e2e/scaffold/apisix.go

```diff
@@ -46,7 +46,7 @@ spec:
       app: apisix-deployment-e2e-test
   strategy:
     rollingUpdate:
-      maxSurge: 50%
+      maxSurge: 50%%
```

Comment:
typo？

Reply:
No, if you want to output a string using format characters, you need to use %% to represent the percentage sign.
![image](https://github.com/apache/apisix-ingress-controller/assets/79972061/92cb98c6-76df-4d36-92c6-f81a87e80eef)